src/bitops.c

```diff
@@ -411,6 +411,16 @@ void printBits(unsigned char *p, unsigned long count) {
 #define BITOP_OR    1
 #define BITOP_XOR   2
 #define BITOP_NOT   3
+#define BITOP_DIFF  4 /* DIFF(X, A1, A2, ..., An) = X & !(A1 | A2 | ... | An) */
+#define BITOP_DIFF1 5 /* DIFF1(X, A1, A2, ..., An) = !X & (A1 | A2 | ... | An) */
+#define BITOP_ANDOR 6 /* ANDOR(X, A1, A2, ..., An) = X & (A1 | A2 | ... | An) *//* */
+
+/* ONE(A1, A2, ..., An) = X.
+ * If X[i] is the i-th bit of X then:
+ * X[i] == 1 iff there is m such that:
```

Comment:
```suggestion
 * X[i] == 1 if there is m such that:
```

Reply:
btw iff can be often found in papers meaning `if and only if`. Will expand it.